pkg/config/plan.go

```diff
@@ -56,8 +64,8 @@ func (plan *Plan) GetPlan(filename string) (*Plan, error) {
 }
 
 func validate(plan *Plan) error {
-	if plan.Bucket.S3.Name != "" && plan.Bucket.GS.Name != "" {
-		return errors.New("error in configuration : should only have s3 OR gs bucket configured")
+	if plan.Bucket.S3.Name != "" && plan.Bucket.GS.Name != "" && plan.Bucket.Minio.Name != "" {
```

Comment:
Not blocker, but it will pass the condition if two buckets are defined (true && false && true == false) are defined, and wont block the customer.
As it is an extra check (and the first configuration will be taken, it can be left).